client/app.js

```diff
@@ -0,0 +1,35 @@
+let form = document.getElementById("form");
+
+form.addEventListener("submit", handler)
+
+function handler (e) {
+    e.preventDefault();
+    const articleData = {
+        title: e.target.title.value,
+        user: e.target.author.value,
+        article: e.target.article.value,
+        //ts: Date.now(),
+    };
+
+    const options ={
+            method: 'POST',
+            body: JSON.stringify(articleData)
+    };
+
+    fetch("http://localhost:3000/post", options)
+            .then(res => (res).json())
+            .then(() => appendArticle(articleData))
+            .catch((error) => {
+                console.error('Error:', error)
+            })  
+}
+
+// function appendArticles(data){
+//     data.cats.forEach(appendArticle);
+// };
+
+function appendArticle(articleData){
```

Comment:
entering the new articles into the document is a good idea. but this way would the users be able to see all the others posts as well?